### DISCUSSION TITLE
Some windows remain in Taskbar after being closed

### DISCUSSION THREAD

**Discussion by tomasss1024** [lawand-dot-io/taskbar#106] [Feature requests]
If you close one window from a multi-window application, the taskbar window remains displayed.
Example: Open new window from Firefox or Edge (Com+N) and close only this new window (red button). The window is still on taskbar.

**Comment by lawand-dot-io** [comment]
Thanks for the report. What version of macOS are you using?

**Reply by tomasss1024** [reply]
Hello, it's Sequoia Version 15.1.1 (24B91). FF v132.0.2, Edge 130.0.2849.46. 
This problem is also on Double Commander 1.1.20 (view some files via F3 and close)

**Reply by lawand-dot-io** [reply]
Thanks, yeah there's a bug detecting window closure on macOS 15. I am working on a solution.

**Comment by lawand-dot-io** [comment]
Will close this ticket and continue posting updates on https://github.com/lawand-dot-io/taskbar/issues/118
